Description: Add --setuptools as an explicit option in virtualenv.py.
 This also allows the use of $VIRTUALENV_SETUPTOOLS environment variable to
 select setuptools over distribute.  This patch does not change the default
 (see next patch in series for that).
Origin: https://github.com/warsaw/virtualenv/compare/develop...setuptools
Author: Barry Warsaw <barry@python.org>

diff --git a/virtualenv.py b/virtualenv.py
index 8fdb275..92f48b9 100755
--- a/virtualenv.py
+++ b/virtualenv.py
@@ -858,6 +858,16 @@ def main():
         help='Use Distribute instead of Setuptools. Set environ variable '
         'VIRTUALENV_DISTRIBUTE to make it the default ')
 
+    parser.add_option(
+        '--setuptools', 
+        dest='use_distribute', 
+        action='store_false',
+        help='Use Setuptools instead of Distribute.  Set environ variable '
+        'VIRTUALENV_SETUPTOOLS to make it the default ')
+
+    # Set this to True to use distribute by default, even in Python 2.
+    parser.set_defaults(use_distribute=False)
+
     default_search_dirs = file_search_dirs()
     parser.add_option(
         '--extra-search-dir',
